Makefile

```diff
@@ -1201,6 +1201,8 @@ horizon_kuttl_run: ## runs kuttl tests for the horizon operator, assumes that ev
 
 .PHONY: horizon_kuttl
 horizon_kuttl: namespace input openstack_crds openstack_storage_crds deploy_cleanup mariadb mariadb_deploy keystone keystone_deploy infra horizon_deploy_prep horizon ## runs kuttl tests for the horizon operator. Installs openstack and openstack-storage crds, mariadb, keystone and horizon operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
+	timeout 120s bash -c 'until [ "$(oc get deployment -l openstack.org/operator-name: horizon -n ${NAMESPACE} -o name)" != "" ]; do sleep 1; done'
+	oc wait deployment -l openstack.org/operator-name: horizon -n ${NAMESPACE} --for condition=Available --timeout=120s
```

Comment:
This pattern is almost available across all operators.  Just waiting for these two reviews and then we can update the label used by `make wait`:

https://github.com/openstack-k8s-operators/cinder-operator/pull/204
https://github.com/openstack-k8s-operators/keystone-operator/pull/245